docs/architecture/README.md

```diff
@@ -82,7 +82,9 @@ This document specifies the architecture for GCP Ingestion as a whole.
    1. Resolve GeoIP from `remote_addr` or `x_forwarded_for` attribute into
       `geo_*` attributes
    1. Parse `agent` attribute into `user_agent_*` attributes
-   1. Copy attributes to `metadata` top level key in `payload`
+   1. Produced `normalized_` variants of select attributes
+   1. Inject `normalized_` attributes at the top level and other select
+      attributes into a nested `metadata` top level key in `payload`
```

Comment:
We generally use top-level as a compound adjective in this document (though notably in the previous line it is not).
```suggestion
      attributes into a nested `metadata` top-level key in `payload`
```